content/graphing/infrastructure/cloudfunctions.md

```diff
@@ -0,0 +1,125 @@
+
+Ensure you have the `tag:GetResources` permission in your Datadog IAM policy in order to have custom tags collected.
+
+### String Search
+
+Search for functions using flexible string search that matches substrings on function name, AWS Metadata, and custom tags.
+
+### Filtering
+
+Use the faceted search functionality along the left-hand side of the page to narrow down the functions that are in view. All AWS and custom tags are available to use as filters.
+
+### Selecting metrics on table
+
+You can check and uncheck metrics you would like to view on the functions table. By default, invocations, duration (avg), and errors are on by default.
+
+Non-default metrics you can check are
+
+- ConcurrentExecutions
+- Throttles
+- IteratorAge
+- DeadLetterError
+- Memory Used (avg)
+- Last Start
+- Estimated Cost
+
+## Function Detail view
+
+Clicking on a particular function in the function summary table brings you to a function detail page. This page provides detailed trace and log level information for that function.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-functiondetailview.png" alt="Cloud Functions - Function Detail View" responsive="true" style="width:80%;">}}
+
+### Summary graphs and time selector
+
+Use the summary graphs across the top of screen and the time selector to focus in on a specific time frame you are interested in. The entire page, including the traces and logs shown, updates when you use the time selector.
+
+### Traces
+
+Via the AWS X-Ray integration, traces from the function currently being viewed are shown in the `Traces` tab. You can sort these traces by attributes such as date, duration, and status.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-traces.png" alt="Traces" responsive="true" style="width:80%;">}}
+
+#### Trace detail view
+
+Clicking on a particular trace opens the trace detail view for that trace. The X-Ray subsegments are transformed into Datadog spans while preserving the naming paradigms, span metadata, and structure of the overall trace.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-traces2.png" alt="Traces" responsive="true" style="width:80%;">}}
+
+Datadog provides specially formatted serverless traces for readability and usability. Clicking on the span from another Lambda function creates a link to that function’s detail page so you can easily jump to another function that is part of the trace.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-traces3.gif" alt="Traces" responsive="true" style="width:80%;">}}
+
+### Logs
+
+All logs emitted from that function are pulled into the function detail page as well. Narrow down the timeframe of the page to a specific moment of interest to view the logs during a critical point in time. Click on the logs in the table to see the full log in more detail.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-logs.png" alt="logs" responsive="true" style="width:80%;">}}
+
```

Comment:
Can we have a further reading section here?

1) X-ray doc
2) ... something else